src/ECER.Managers.Registry.Contract/Applications/Contract.cs

```diff
@@ -33,8 +33,14 @@ public record Application(string? Id, string RegistrantId, ApplicationStatus Sta
   public DateTime? CreatedOn { get; set; }
   public DateTime? SignedDate { get; set; }
   public IEnumerable<CertificationType> CertificationTypes { get; set; } = Array.Empty<CertificationType>();
+  public IEnumerable<Transcript> Transcripts { get; set; } = Array.Empty<Transcript>();
   public PortalStage Stage { get; set; }
 }
+public record Transcript(string? Id, string? EducationalInstitutionName, string? ProgramName, string? StudentName, string? StudentNumber, DateTime StartDate, DateTime EndDate)
```

Comment:
the nullable properties can probably move down as properties; is there a reason to make some properties in the ctor and some outside the ctor?